Adhoc/sales agent docs (#1494)
shopware/docs: products/sales-agent/users.md

---
nav:
   title: User creation
   position: 60

---

# Creating a user and set restrictions

You can create a user by sending a POST request to the `/api/user` endpoint. The x-api-key header is required and can be set in the environment (`API_AUTH_SECRET_KEY`).

```bash
curl -XPOST -H 'x-api-key: <secret>' -H "Content-type: application/json" -d '{
    "name": <name>,
    "email": <email>,
    "id": <id>,
    "password": <password>
}' '<sales_agent_instance_url>/api/user'
```

> If you want to restrict the user so that they can only see specific customers, you can send a POST request to the `/api/entity-restriction` endpoint.

Reviewer: ```suggestion
To restrict a user to viewing only specific customers, you can send a `POST` request to the `/api/entity-restriction` endpoint.
```
